### ISSUE TITLE
Copying wikitext is not enough for files storing all metadata in SDC

### ISSUE THREAD

**Issue by jarek-tuszynski** [danmichaelo/croptool#207]
I was just fixing [File:IAPH-Paisaje_cultural_de_Montoro-PHOTO-2024-05-07-12-38-45_(cropped).jpg](https://commons.wikimedia.org/wiki/File:IAPH-Paisaje_cultural_de_Montoro-PHOTO-2024-05-07-12-38-45_(cropped).jpg) where page created by CropTool was missing a license and had some Lua Errors. The issue was that the source file was storing license data in SDC and was using {{license from structured data}} template to create license info. CropTool copied the wikitext but not SDC data leaving the crop file without metadata. CropTool should be copying some or all SDC metadata from source file.  If any help is needed on Commons please let me know by pinging "Jarekt"

**Comment by danmichaelo** [comment]
@jarek-tuszynski Right, I'm not really familiar with SDC at all. Seems like there are many fields that should not be copied over (width, height, size, checksum, etc.). Would you recommend creating an include list of properties to copy over (license, etc.) or an exclude list or something smarter?

**Comment by don-vip** [comment]
I guess an exclude list would be fine. I think about this one:
- P2048 (height)
- P2049 (width)
- P3575 (data size)
- P4092 (checksum)
- P9310 (pHash checksum)

**Comment by lubianat** [comment]
Hi! 

I opened a small discussion on the Talk page on this exact topic: 

https://commons.wikimedia.org/wiki/Commons_talk:CropTool#Structured_Data

I think P18 (depicts) is significant enough to warrant some kind of check (or at least a note/warning), because it will not be the case some times.

**Comment by hayleox** [comment]
Just wanted to boost this – I was planning to start uploading photos with only SDC, but losing CropTool compatibility is a dealbreaker for me.

In addition to copying the existing statements, CropTool should probably also add a [P12346](https://www.wikidata.org/wiki/Property:P12346) (based on media) claim pointing back to the original file.

Re @lubianat: This is not an SDC-specific problem -- the description/categories in the wikitext of images cropped with CropTool is also often wrong. I think CropTool should just copy over the P18 claims as-is for now; making sure users are paying attention to metadata is a separate concern.
